.github/workflows/screenshot.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
     steps:
     - name: Check out the code
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
```

Comment:
Undocumented and otherwise not in scope for a PR ostensibly about typos?

Reply:
I reverted this change.
I will come up with this change in another PR once this PR is merged.